app/exporters/odt/cv.rb

```diff
@@ -8,10 +8,11 @@ def initialize(person)
       @person = person
     end
 
-    def export
-      ODFReport::Report.new('lib/templates/cv_template.odt') do |r|
-        insert_general_sections(r)
-        insert_personalien(r)
+    # rubocop:disable Metrics/LineLength
+    def export(anon = true)
+      ODFReport::Report.new(anon ? 'lib/templates/cv_template_anon.odt' : 'lib/templates/cv_template.odt') do |r|
```

Comment:
Here you can use a variable again. Also, the path stays the same... just concat the path string with the correct filename